src/actions/navigation/index.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {NavigationTypes} from 'constants';
+import Routes from 'navigation/routes';
+
+export function goBack() {
+    return async (dispatch, getState) => {
+        dispatch({
+            type: NavigationTypes.NAVIGATION_POP
+        }, getState);
+    };
+}
+
+export function goToLogin() {
+    return async (dispatch, getState) => {
+        dispatch({
+            type: NavigationTypes.NAVIGATION_PUSH,
+            route: Routes.Login
+        }, getState);
+    };
+}
+
+export function goToSelectTeam() {
+    return async (dispatch, getState) => {
+        dispatch({
+            type: NavigationTypes.NAVIGATION_PUSH,
+            route: Routes.SelectTeam
+        }, getState);
+    };
+}
+
+export function goToChannel() {
```

Comment:
Right now, it's just switching to the scene named Channel, not to a particular channel. I'll leave it as it is for now, but I'll try to come up with a better name for it. We don't really have a better name for the main view/center channel/channel view right now, but we do need one to avoid ambiguity